lib/routes/sign.js

```diff
@@ -2,6 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var crypto = require('crypto')
+
+var TIMED_KEY = crypto.randomBytes(8)
+setInterval(function () {
+  TIMED_KEY = crypto.randomBytes(8)
+}, 1000 * 60 * 60 * 24);
```

Comment:
e.g. an alternative is to just aggregate our counts on a daily basis and then throw the dat away